command/exec_test.go

```diff
@@ -306,6 +307,39 @@ func (s *execCmdSuite) TestKeepEmptyArgs() {
 	s.Len(cmd.Args, 2)
 }
 
+func (s *execCmdSuite) TestPathSetting() {
+	cmd := &subprocessExec{
+		// just set up enough so that we don't fail parse params
+		Env:        map[string]string{},
+		WorkingDir: testutil.GetDirectoryOfFile(),
+		Path:       []string{"foo", "bar"},
+	}
+	exp := util.NewExpansions(map[string]string{})
+	s.Len(cmd.Env, 0)
+	s.NoError(cmd.doExpansions(exp))
+	s.Len(cmd.Env, 1)
+
+	path, ok := cmd.Env["PATH"]
+	s.True(ok)
+	s.Len(filepath.SplitList(path), 2)
```

Comment:
Shouldn't this be 3? (what already exists in path, plus "foo", plus "bar"

Reply:
it's probably way more than three (because the value of `$PATH` is expanded) but let's see what the tests say, but I'll probably change it to len(path) >= 2)